irc/server.go

```diff
@@ -72,6 +73,10 @@ func NewServer(config *Config) *Server {
 		server.listen(addr)
 	}
 
+	if config.Server.Wslisten != "" {
```

Comment:
For consistency, do you think we should allow multiple `wslisten` like we do for `listen`?